docs/en/Mounting-Alluxio-FS-with-FUSE.md

```diff
@@ -35,70 +27,68 @@ After having properly configured and started the alluxio cluster, and from the n
+```bash
+$ integration/fuse/bin/alluxio-fuse umount /mnt/people
+Unmount fuse at /mnt/people (PID: 97626).
+```
 
-1. The base directory of `pathname` exists in Alluxio;
-2. `O_CREAT` and `O_WRONLY` are passed among the `flags` bitfield.
+### Check the Alluxio-FUSE mounting status
 
-Equivalently, `creat(const char* pathname )` calls will succeed as long as (1) holds and `pathname` does not exist yet.
+{% include Mounting-Alluxio-FS-with-FUSE/alluxio-fuse-stat.md %}
 
-If `pathname`, instead, points to an existing regular file in Alluxio, then an open call will only succeed if:
+This outputs the `pid, mount_point, alluxio_path` of all the running alluxio-fuse processes.
 
-1. `O_RDONLY` is passed among the `flags` bitfield.
+For example, the output will be like:
 
-Note that, in either cases, the `mode` parameter is currently ignored by Alluxio-FUSE.
+```bash
+pid	mount_point	alluxio_path
+80846	/mnt/people	/people
+80847	/mnt/sales	/sales
+```
 
-### `read(int fd, void* buf, size_t count)` (see also `man 2 read`)
+## Optional configuration steps
 
-A read system call will only succeed when `fd` refers to an Alluxio file that has been previously opened with the `O_RDONLY` flags.
+Alluxio-FUSE is based on the standard Java client API `alluxio-core-client-fs` to perform its operations. You
+might want to customize the behaviour of the alluxio client used by Alluxio-FUSE the same way you
+would for any other client application.
 
-### `lseek(int fd, off_t off, int whence)` (see also `man 2 lseek`)
+One possibility, for example, is to edit `$ALLUXIO_HOME/conf/alluxio-site.properties` and set your specific alluxio client options.
 
-Seeking is supported only on files open for reading, i.e., on files that have been opened with an `O_RDONLY` flag.
+## Assumptions and limitations
 
-### `write(int fd, const void* buf, size_t count)` (see also `man 2 write`)
+Currently, most basic file system operations are supported. However, due to Alluxio implicit characteristics, please be aware that:
 
-A write system call will only succeed when `fd` refers to an Alluxio file that has been previously
-opened  with the `O_WRONLY` flag.
+* Files can be written only once, only sequentially, and never be modified. That means overriding a file is allowed, and an explicit combination of delete and then create is needed. For example, `cp` command will fail when the destination file exists. 
+* Alluxio does not have hard-link and soft-link concepts, so the commands like `ln` are not supported, neither the hardlinks number is displayed in `ll` output.
+* The user and group are mapped to the Unix user and group only when Alluxio is configured to use shell-based mapping, by setting `alluxio.security.group.mapping.class` to `ShellBasedUnixGroupsMapping` in `conf/alluxio-site.properties`. Otherwise `chown` and `chgrp` are no-ops, and `ll` will return the user and group as the one who started the alluxio-fuse process.
```

Comment:
"... and `ll` will return the user and group of the user who started the alluxio-fuse process"

Reply:
changed. thanks.